frameworks/JavaScript/nodejs/create-server.js

```diff
@@ -0,0 +1,84 @@
+// Forked workers will run this code when found to not be
+// the master of the cluster.
+
+var http = require('http');
+var url = require('url');
+var h = require('./helper');
+
+// Handlers, one for each db config
+var MongodbRawHandler = require('./handlers/mongodb-raw');
+var MySQLRawHandler = require('./handlers/mysql-raw');
+
+// Mongoose is a popular Node/MongoDB driver
+var MongooseHandler = require('./handlers/mongoose');
+
+// Sequelize is a popular Node/SQL driver
+var SequelizeHandler = require('./handlers/sequelize');
+
+// Node's redis package uses the C bindings of the hiredis library
+var HiredisHandler = require('./handlers/redis');
+
+module.exports = http.createServer(function (req, res) {
+  var values = url.parse(req.url, true);
```

Comment:
`url` module is pretty big and complicated, not my first choice. You could try `parseurl` https://github.com/pillarjs/parseurl (used in express, don't judge by github stars) or even a simple regexp, as you don't use the vast output generated by url.parse